de-lib/src/main/java/org/iplantc/de/theme/base/client/fileViewers/FileViewerContextualHelpStrings.properties

```diff
@@ -1 +1,8 @@
-sendToEnsemblUrlHelp = To access this file in genome browsers, this file needs to be public. If you do not want this file to be public, you can change its permissions as follows: <b>1.</b> Navigate to the file in the data window <b>2.</b> Select the file <b>3.</b> Select <i>Share</i>-><i>Share with collaborators</i> from the menu. <b>4.</b> In the <i>Manage Sharing</i> window in the <i>Who has access</i> section and remove permissions to the <i>anonymous</i> user.
+sendToEnsemblUrlHelp = To access a file in genome browsers, the file needs to be public. \
+  If you do not want the file to be public, you can change its permissions as follows:\
+  <ol>\
+  <li><b>1.</b> Navigate to the file in the data window.</li>\
+  <li><b>2.</b> Select the file.</li>\
+  <li><b>3.</b> Select <i>Share</i>-><i>Share with collaborators</i> from the menu.</li>\
+  <li><b>4.</b> In the <i>Manage Sharing</i> window in the <i>Who has access</i> section and remove permissions to the <i>anonymous</i> user.</li>\
```

Comment:
The help text won't display the HTML for these `li` items, unfortunately, so the numbers are still required as part of the text; but at least it will display each item with line breaks in between them.